Utils/Test-RscSdk.ps1

```diff
@@ -49,6 +50,9 @@ Set-Location $PSScriptRoot\..
 # Stop on error
 $ErrorActionPreference = "Stop"
 
+# Uncomment this to enable Write-Debug output
+# $DebugPreference = "Continue"
```

Comment:
would we be able to turn this into a parameter?

Reply:
I tried lol 
this is still confusing to me . `-Debug` is a standard cmdlet parameter, but it still needs the `$DebugPreference` to be set.